api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -54,8 +59,34 @@ def assertProjectData(self, project):
         self.assertIsNotNone(sample.tissue_location)
         self.assertIsNotNone(sample.treatment)
 
-    def assertProjectReadmeContains(self, text, project_zip):
-        self.assertIn(text, project_zip.read("README.md").decode("utf-8"))
+    def assertProjectReadmeContent(self, zip_file, project_id=""):
+        def get_updated_content(content):
+            """
+            Replace the placeholders PROJECT_ID and TEST_TODAYS_DATE in test_data/readmes
+            with the given project_id and today's date respectively for format testing."
+            """
+            content = re.sub("PROJECT_ID", project_id, content)
```

Comment:
We should loop over the project ids so we probably want to zero index them. ie  `PROJECT_ID_0` in the template to make it easier.

Reply:
See at 15f2982183f55ca8734740d5ab5c7db6b6d452b1.